### ISSUE TITLE
Dropdown in tiles

### ISSUE THREAD

**Issue by Aaarrrgggh** [Spiderpig86/Cirrus#50]
Hi!

Thanks for your cool framework!

I tried it on my project, and I noticed some bugs (or maybe, I don't use correctly the HTML code):

I would like to use dropdown buttons in a tile. Seems to work on desktop, but there are some display issues on mobile. Because dropdown block contains the ul menu, the area goes outside the screen.

Thanks !

Alexander

**Comment by Spiderpig86** [comment]
Hi,

Can you please provide more information on this issue? I recommend starting with the issue template on the repository.

```
**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
```

It would help if could provide the code as well.

Thanks.

**Comment by Aaarrrgggh** [comment]
**Bug**
Bug happens when we want to use dropdown buttons in tiles (button container, right to the screen). If we don't change the min-width of menu element, display is broken (on mobile phones only)

**Reproduce**
Any tile example and any dropdown list.

Here, the code I used:


```
<section class="row">
<div class="tile">
  <div class="tile__icon">
  <figure class="avatar"><img src="https://www.seoclerk.com/pics/319222-1IvI0s1421931178.png"></figure>
  </div>
  <div class="tile__container">
  <p class="tile__title m-0">Robert Downey Jr. shared a post from <b>Stark Industries</b>.</p>
  <p class="tile__subtitle m-0">Robert shared: 'Stark Industries is proud to announce its brand new suit.'</p><span class="info">23 minutes ago</span></div>
  <div class="tile__buttons m-0 u-text-right">
  <div class="list-dropdown dropdown-right">
  <button class="btn-transparent btn-dropdown m-0">
  <i  class="icon fad fa-ellipsis-v " aria-hidden="true"></i>
  </button>
  <ul class="menu u-text-left">
  <li class="menu-item"><a href="!#"><i  class="icon fad fa-cogs " aria-hidden="true"></i> Settings</a></li>
  <li class="menu-item"><a href="!#"><i  class="icon fad fa-cogs " aria-hidden="true"></i> Firefox</a></li>
  <li class="menu-item"><a href="!#"><i  class="icon fad fa-cogs " aria-hidden="true"></i> Polarity</a></li>
  </ul>
  </div>
  </div>
  </div>
  </section>

```

Hope it helps,

Sincerely

**Comment by Spiderpig86** [comment]
Thanks for the sample code, I'll be making a fix for this in 0.6.1.

**Comment by Spiderpig86** [comment]
I was able to fix this issue with commit 6d87ac19a5b8c777ea24b1768d1a39dd7339a607. I took a look at the code you provided and added the `u-position-absolute` class on the menu itself to achieve the behavior you want.

```html
<div class="tile p-1 bg-gray-200 u-round">
    <div class="tile__icon">
        <figure class="avatar"><img src="https://www.seoclerk.com/pics/319222-1IvI0s1421931178.png" /></figure>
    </div>
    <div class="tile__container">
        <p class="tile__title m-0">Robert Downey Jr. shared a post from <b>Stark Industries</b>.</p>
        <p class="tile__subtitle m-0">Robert shared: 'Stark Industries is proud to announce its brand new suit.'</p>
        <span class="info">23 minutes ago</span>
    </div>
    <div class="tile__buttons m-0 u-text-right">
        <div class="list-dropdown dropdown-right">
            <button class="btn-transparent btn-dropdown m-0">
                <i class="icon fad fa-ellipsis-v" aria-hidden="true"></i>
            </button>
            <ul class="menu u-text-left u-position-absolute">
                <li class="menu-item">
                    <a href="!#"><i class="icon fad fa-cogs" aria-hidden="true"></i> Settings</a>
                </li>
                <li class="menu-item">
                    <a href="!#"><i class="icon fad fa-cogs" aria-hidden="true"></i> Firefox</a>
                </li>
                <li class="menu-item">
                    <a href="!#"><i class="icon fad fa-cogs" aria-hidden="true"></i> Polarity</a>
                </li>
            </ul>
        </div>
    </div>
</div>
```
